package.json

```diff
@@ -42,6 +42,6 @@
     ]
   },
   "devDependencies": {
-    "eslint": "^6.7.2"
```

Comment:
why did you remove eslint

Reply:
conflict with standard

Reply:
okay. so you only catch the linting errors once you run the standard command @shaunorpen